tests/neg-macros/i6432b.check

```diff
@@ -1,6 +1,42 @@
-[63..66] in Test_2.scala
-fgh
-[52..55] in Test_2.scala
-xyz
-[41..44] in Test_2.scala
-abc
+checking tests/neg-macros/i6432b/Macro_1.scala after phase frontend
+checking tests/neg-macros/i6432b/Macro_1.scala after phase inlinedPositions
+checking tests/neg-macros/i6432b/Macro_1.scala after phase staging
+checking tests/neg-macros/i6432b/Macro_1.scala after phase posttyper
+checking tests/neg-macros/i6432b/Macro_1.scala after phase pickler
+checking tests/neg-macros/i6432b/Macro_1.scala after phase reifyQuotes
+checking tests/neg-macros/i6432b/Macro_1.scala after phase MegaPhase{firstTransform, checkReentrant, elimPackagePrefixes, cookComments}
```

Comment:
All these checking lines should be filtered out

Reply:
I agree that these don't look very nice. However, they come from the `-Ycheck:all` compiler flag with which the tests are compiled. If we take it seriously that we want an exact console output in these check files, these lines are the correct behaviour. If we don't want them, we should run the tests without the `-Ycheck:all` flag. What does that flag do? Do you think we can drop it?

Reply:
That is the most important testing flag. It must never be dropped. It runs a bunch of checks between each phase to make sure the trees are well formed and typed.

Reply:
Fixed